Staging_Dev/20190401T202900Z_welcome_mail.sql

```diff
@@ -0,0 +1,46 @@
+USE [perpetuumsa]
+GO
+
+/* Setup premademail */
+create table dbo.premademail (
+	id int primary key,
+	subject nvarchar(128) not null,
```

Comment:
Let's consider a unique name field here and do the lookup server-side by this name, so order of insertion won't matter.
Then update https://github.com/OpenPerpetuum/PerpetuumServer/pull/98/files#diff-978e4c8223e0060f4ff0d26e11bf5c24R196 and the other calls to lookup messages by a semantically clear and unique short-name of some kind.